DigitalLearningSolutions.Data.Tests/Services/CourseContentServiceTests.cs

```diff
@@ -445,6 +445,7 @@ public void Get_course_content_should_only_return_sections_with_tutorials_in_cus
         [TestCase(1, 15853, 101, 173218)]
         [TestCase(254480, 24224, 101, null)]
         [TestCase(22044, 10059, 121, 100467)]
+        [TestCase(11, 6226, 101, 89490)]

```

Comment:
Great, thank you! Do keep adding any course you think might be useful to this test, the more the better! 🙂

Reply:
Will do 🙂